bazel/cc_toolchains/clang_cc_toolchain_config.bzl

```diff
@@ -152,10 +152,14 @@ def _impl(ctx):
                 actions = all_cpp_compile_actions + all_link_actions,
                 flag_groups = ([
                     flag_group(
-                        flags = [
-                            "-std=c++17",
-                            "-stdlib=libc++",
-                        ],
+                        flags = ["-std=c++17"] if ctx.attr.target_cpu == "x64_windows" else ["-std=c++17", "-stdlib=libc++"]
```

Comment:
Echoing my prior comment, can you please avoid use of Python's ternary operator for readability? Instead do something like:

```
compile_flags = ["-std=c++17"]
# libc++ is only used on non-Windows platforms.
if target_cpu != "x64_windows":
  compile_flags.append("-stdlib=libc++")
```

Then here: `flags = compile_flags`

I'd suggest this as code edits, but I can't do a code edit suggestion away from where you're making changes, and I think the compile_flags should go around line 99.